ome_zarr/format.py

```diff
@@ -69,6 +69,9 @@ def _get_multiscale_version(self, metadata: dict) -> Optional[str]:
         if multiscales:
             dataset = multiscales[0]
             return dataset.get("version", None)
+        plate = metadata.get("plate", [])
```

Comment:
Should the method be renamed to match the fact that it detects the version outside the scope of the `multiscales` specification or do you see a driver for keeping separate `_get_<spec>_version`?

Also will this method need to be extended to support every intermediate hierarchy level that would define a version, for instance `well` ?

Reply:
Good catch! Fixed in [d1b93bc](https://github.com/ome/ome-zarr-py/pull/195/commits/d1b93bc56cdc679073873e696da5fb45e3569c6f)
This now allows viewing a Well (which didn't work before) e.g.
```
$ napari --plugin napari-ome-zarr https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.1/plates/422.zarr/A/1 -vvv
```